website/useful_tools.rst

```diff
@@ -0,0 +1,63 @@
+===================
+Useful Python Tools
+===================
+
+This is a list of useful and/or new Python tools that the Unidata Python Team
+and community are keeping an eye on or using.
+
+----------------
+Unidata Projects
+----------------
+* MetPy_ - General meteorological toolkit
+* siphon_ - Remote data access
+* netCDF4-python_ - netCDF4 API
+
+--------------------
+Meteorology Specific
+--------------------
+* PyART_ - Python ARM Radar Toolkit
+
+--------------
+Data Wrangling
+--------------
+* pandas_ - Easy tabular data manipulation
+* xarray_ - Gridded/labeled multidimensional data
+
+-----------
+Performance
+-----------
+* Numba_ - JIT compiler
+* Dask_ - Distributed computing
+
+--------
+Plotting
+--------
+* matplotlib_ - Beautiful publication quality graphics
+* Bokeh_ - Interactive web graphics
```

Comment:
Do we have to include Bokeh?

Reply:
Also, need CartoPy

Reply:
Fair enough. Just letting my matplotlib colors show through.

Reply:
I know, you bleed viridis